cloudsync/sync/manager.py

```diff
@@ -154,7 +154,7 @@ def __init__(self, state: SyncState,                        # pylint: disable=to
         self.translate = lambda side, path: self.__translate(side, path) if path else None
         self._resolve_conflict = resolve_conflict
         self.tempdir = tempfile.mkdtemp(suffix=".cloudsync")
-        self.__nmgr = notification_manager
+        self._nmgr = notification_manager
```

Comment:
I'm not saying that making __nmgr more public in the SyncManager is a problem, but it doesn't seem to be necessary for this PR? the nmgr that is used in smartsync is the CloudSync class not the SyncManager class

Reply:
Ya you're right it's not strictly necessary any more. The change that went in to call the `SyncManager`'s init method from `SmartSyncManager` fixed the problem we were seeing last week with the `__nmgr` name mangling. Although I do think this change generally makes sense if smartsync is going to be inheriting from this class and may want to access the nmgr.